mouse: Add acceleration profiles

vkareh opened the pull request:
Currently libinput supports Adaptive and Flat acceleration profiles. We
can use these to change mouse behavior. Synaptic touchpads are not
supported through libinput, so they will not be affected by this change.

There does not seem to be a defined behavior for a Default acceleration
profile, so it's not included.

Multi-part patch:
* https://github.com/mate-desktop/mate-desktop/pull/407
* https://github.com/mate-desktop/mate-settings-daemon/pull/293
* https://github.com/mate-desktop/mate-control-center/pull/494

monsta commented on data/org.mate.peripherals-touchpad.gschema.xml.in:
```diff
@@ -85,5 +89,10 @@
+    <key name="accel-profile" enum="org.mate.peripherals-touchpad.AccelProfile">
+      <default>'adaptive'</default>
+      <summary>Acceleration profile</summary>
+      <description>Acceleration profile for touchpad. The acceleration profile can be either 'adaptive', which adapts the acceleration depending on the mouse movement, or 'flat, which accelerates by a device-specific constant factor derived from the configured pointer speed.</description>
```
Small thing: missing single quote after "flat" in description.

vkareh commented:
@raveit65, @monsta - I've moved the touchpad gschema to mate-desktop. I've also added a `'default'` acceleration profile, which reads the libinput default and sets it as the current acceleration profile.

raveit65 commented:
https://travis-ci.org/mate-desktop/mate-settings-daemon/jobs/570270196#L2716
`NEWS not updated; not releasing`  :)

@vkareh
We are using the `make distcheck` command at the end of travis builds.
You need to bump version in NEWS file too.
Simply add an new empty entry, we only need the version in NEWS for `make distcheck`.
```
### mate-settings-daemon 1.23.1

### mate-settings-daemon 1.23.0

    tx: sync with transifex
    media-keys: Execute default instant messaging application defined by schema
    Change url project's website
    locate-pointer: Do not grab pointer button when trying to locate
    Create FUNDING.yml
    [ci] Add cppcheck html report
    [ci] Enable Clang Static Analyzer
```

raveit65 commented:
I fixed the missing version bump in NEWS, so it should build fine at travis now.

raveit65 commented:
Ok, than merging this PR will make a new release. :)
Normally i write the NEWS when i make the release + making the release tag.
For this reason i added an empty entry in NEWS.
Anyway, it's OK , no need to revert it ;)

raveit65 commented:
LGTM,
after updadating all 3 packages it works like expected and can change acceleration profiles in mate-mouse-properties.
Are there any concerns left?
Or can i merge it and make a new release for fixing m-c-c PR?
Keep in mind that we currently don't have mouse and keyboard schemes in git repos after new mate-desktop releases.
